extendr-api/src/metadata.rs

```diff
@@ -99,13 +93,24 @@ fn write_r_wrapper(w: &mut Vec<u8>, func: &Func, package_name: &str, use_symbols
         }
         writeln!(w, "")?;
     }
-    
-    write!(w, "{}{}{} <- function({}) .Call(", dest_prefix, dest_sep, func.name, args)?;
+    Ok(())
+}
+
+fn write_function_wrapper(w: &mut Vec<u8>, func: &Func, package_name: &str, use_symbols: bool) -> std::io::Result<()> {
+    if func.hidden {
+        return Ok(());
+    }
+
+    write_doc(w, func.doc)?;
+
+    let args = func.args.iter().map(|arg| arg.name).collect::<Vec<_>>().join(", ");
```

Comment:
Because of #125, is it possible to input an additional check here?
From Rust guidelines (if I understand correctly) https://doc.rust-lang.org/reference/identifiers.html,
valid parameter names can start with the underscore `_` (which is not useful, but allowed).
If such parameter name is emitted into R, the resulting R code is invalid, as R does not allow identifiers to start with `_`.
Can we replace map with a function that matches first character and if it is `_`, wraps name into backquotes, like ``` `_y` ```.

Here is my [improvised attempt](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=736766f593c7651f144ad387707df182).